install_deps.sh

```diff
@@ -54,101 +54,113 @@ fi
+            esac
+        done
+    fi
 }
 
+
 ### Main ###
 
 if [ "${ID_LIKE}" = debian ] || [ "${ID}" = debian ]
 then
-	# For system-wide installs that are not possible via apt
-	temp_dir=/tmp/ILLIXR_deps
-	mkdir -p "${temp_dir}"
-
-	# For local installs
-	opt_dir=/opt/ILLIXR
-	sudo mkdir -p "${opt_dir}"
-	sudo chown $USER: "${opt_dir}"
-
-	if y_or_n "Next: Add apt-get sources list/keys and install necessary packages"; then
-		. ./scripts/install_apt_deps.sh
-	fi
-
-	if [ ! -d "${temp_dir}/opencv" ] && y_or_n "Next: Install OpenCV from source"; then
-		. ./scripts/install_opencv.sh
-	fi
-
-	if [ ! -d "${temp_dir}/Vulkan-Headers" ] && y_or_n "Next: Install Vulkan Headers from source"; then
-		. ./scripts/install_vulkan_headers.sh
-	fi
-
-	if [ ! -d "${opt_dir}/googletest" ] && y_or_n "Next: Install gtest"; then
-		. ./scripts/install_gtest.sh
-	fi
-
-	if [ ! -d "${opt_dir}/qemu" ] && y_or_n "Next: Install qemu (not necessary for core ILLIXR; necessary for virtualization)"; then
-		. ./scripts/install_qemu.sh
-	fi
-
-	# if [ ! -d Vulkan-Loader ]; then
-	# 	echo "Next: Install Vulkan Loader from source"
-	# 	if y_or_n; then
-	# 		git clone https://github.com/KhronosGroup/Vulkan-Loader.git
-	# 		mkdir -p Vulkan-Headers/build && cd Vulkan-Headers/build
-	# 		../scripts/update_deps.py
-	# 		cmake -C helper.cmake ..
-	# 		cmake --build .
-	# 		cd ../..
-	# 	fi
-	# fi
-
-	if [ ! -d "${temp_dir}/OpenXR-SDK" ] && y_or_n "Next: Install OpenXR SDK from source"; then
-		. ./scripts/install_openxr.sh
-	fi
-
-	if [ ! -d "${opt_dir}/gtsam" ] && y_or_n "Next: Install gtsam from source"; then
-		. ./scripts/install_gtsam.sh
-	fi
-
-	if [ ! -d "${opt_dir}/opengv" ] && y_or_n "Next: Install opengv from source"; then
-		. ./scripts/install_opengv.sh
-	fi
-
-	if [ ! -d "${opt_dir}/DBoW2" ] && y_or_n "Next: Install DBoW2 from source"; then
-		. ./scripts/install_dbow2.sh
-	fi
-
-	if [ ! -d "${opt_dir}/Kimera-RPGO" ] && y_or_n "Next: Install Kimera-RPGO from source"; then
-		. ./scripts/install_kimera_rpgo.sh
-	fi
-
-	if ! which conda 2> /dev/null; then
-		if [ ! -d "$HOME/miniconda3" ]; then
-			if y_or_n "Next: Install Conda"; then
-				. ./scripts/install_conda.sh
-			fi
-		fi
-	fi
-
-	# Load new library paths
-	sudo ldconfig
-
-	# I won't ask the user first, because this is not a global installation.
-	# All of this stuff goes into a project-specific venv.
-	$HOME/miniconda3/bin/conda env create --force -f runner/environment.yml
+    # For system-wide installs that are not possible via apt
+    temp_dir=/tmp/ILLIXR_deps
+    mkdir -p "${temp_dir}"
+
+    # For local installs
+    opt_dir=/opt/ILLIXR
+    sudo mkdir -p "${opt_dir}"
+    sudo chown $USER: "${opt_dir}"
+
+    echo "* The user will now be prompted to install the following dependencies and optional features:"
+    echo "  Binary packages (apt-get), Docker, OpenCV, Vulkan, gtest, qemu, OpenXR-SDK, gtsam, opengv, DBoW2, Kimera-RPGO, Conda (miniconda3)"
```

Comment:
Add CUDA to this list.

Reply:
Do we check to see if cuda is already on the machine? I don't see it scrolling through install_deps